static/opta.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-OPTAVERSION=0.19.0
+OPTAVERSION=0.22.0
```

Comment:
is that documented somewhere that we need to update this file whenever we do a release?

Reply:
Added here: https://www.notion.so/runx/Opta-release-runbook-f10ed62ec5db4484ab3a9e7790e1200f

@NitinAgg we have a small step to add to the release process.